cirq-core/cirq/vis/heatmap.py

```diff
@@ -390,13 +410,11 @@ def plot(
         Heatmap({q: 0.0 for q in qubits}).plot(
             ax=ax,
             collection_options={
-                'cmap': 'binary',
```

Comment:
what happens when `selected_qubits` is None ? ... this means that we lose these values, right?

Reply:
Great catch, thanks! I've modified this so that the default is retained if `highlighted_qubits` is `None`.

Reply:
I think this can be done with the original collection_options specification assuming Heatmap.plot uses that for the non-highlighted style (the comment to Heatmap.plot above); the only change here should be adding the highlighted_qubits argument at 428.

Reply:
I understand that ideally we would like to only change the code in `Heatmap.plot()` and only add an argument to the `TwoQubitInteractionHeatmap.plot()` method, but I believe there is a subtlety that prevents this from working in practice.

Your suggestion works seamlessly for single-qubit heatmaps but gives something weird for two-qubit heatmaps:

### With highlight qubits
![download](https://github.com/quantumlib/Cirq/assets/68844397/3a3dcbbe-1fa3-4af4-b07c-dc719227afde)

### Without highlight qubits
![download](https://github.com/quantumlib/Cirq/assets/68844397/f4e710b1-199d-4348-936f-99d5bbf4e45a)

In either case, the inside also gets filled with some value from the viridis colormap somehow.

Also, the code from the `main` branch has this, which was probably there to avoid this behavior:

https://github.com/quantumlib/Cirq/blob/5f7881ffbb84cf107564f8eb5d2c50a297d2e872/cirq-core/cirq/vis/heatmap.py#L390-L397

So perhaps the change that is made here is okay?